### PR TITLE
feat: pass stdin when executing subcommand

### PR DESCRIPTION
**Pull request by moreal**
This pull request makes `StandaloneSubcommand.execSync` to support `stdin` input.

### CODE REVIEW

File: src/main/headless/validation.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { StandaloneSubcommand } from "./subcommand";
 export class Validation extends StandaloneSubcommand {
   public isValidPrivateKey(privateKeyHex: string): boolean {
     try {
-      this.execSync("validation", "private-key", privateKeyHex);
+      this.execSync(["validation", "private-key", privateKeyHex]);
```
Comment (longfin) [inline src/main/headless/validation.ts:7]: Is it intended?

```suggestion
      this.execSync(["validation", "private-key"], privateKeyHex);
```

Comment (moreal) [reply src/main/headless/validation.ts:7]: Yes, it was intended but the code you suggested is correct and right. But `NineChronicles.Headless.Executable validation private-key` doesn't support stdin check. It should be implemented in **planetairum/NineChronicles.Headless** first.

**Review by area363 — APPROVED** [review]
Branch update required!

**Comment by moreal** [comment]
> 
> 
> Branch update required!

@area363 I updated this branch on the latest of the target branch and I updated one more file because the `src/main/headless/apv` command was introduced. Could you review it also please? 🙏🏻

**Comment by area363** [comment]
E2E-Test seems to fail 😢

**Pull request closed without merging**